### DISCUSSION TITLE
[Esame 17 luglio 2024] Esercizio 4

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#583] [Esami M-Z]
![es.4](https://github.com/user-attachments/assets/18f54000-4b35-42fb-80b5-5b5d282aed59)

<details>
  <summary>soluzione del prof</summary>
  <br>
  <img src="https://github.com/user-attachments/assets/8ecf97df-adb6-4cec-a016-0c1bd6eff71a" alt="soluzione"  />

</details>

**Comment by thegeek-sys** [comment]
![immagine](https://github.com/user-attachments/assets/c08903fb-913f-4bc0-98ac-32cab58762cf)

**Comment by niccolo-fato** [comment]
<img width="1163" height="1111" alt="image" src="https://github.com/user-attachments/assets/c05ef882-9d4a-480c-b9ec-bd1d43f70b53" />
